client/components/TestQueueRow/TestQueueRow.css

```diff
@@ -103,6 +103,10 @@ button.more-actions:active {
     font-size: 1.1rem;
 }
 
+.updater-button {
+    max-width: 30%;
+}
```

Comment:
At a larger screen size this is solving the problem of making the button a reasonable size instead of stretching across the entire cell - but on smaller screens the button shrinks too fast!

```
.btn.updater-button {
    width: initial;
}
```

This will accomplish the same task and work on smaller screens too.

I had to add the `.btn` part to override another style.